graphql/mutation_resolver.go

```diff
@@ -123,6 +125,41 @@ func (r *mutationResolver) SetAnnotationMetadataLinks(ctx context.Context, taskI
 	return true, nil
 }
 
+// SaveDistroSection is the resolver for the saveDistroSection field.
+func (r *mutationResolver) SaveDistroSection(ctx context.Context, distroID string, changes *restModel.APIDistro, section distro.DistroSettingsSection, onSave DistroOnSaveOperation) (*DistroWithHostCount, error) {
```

Comment:
I'll defer to an app engineer's opinion on this, but it may be preferable to move the heavy logic out of the resolver/GraphQL utils and into `rest/data/distros.go`